doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,543 @@
+with Roudi.
+
+To do so, the following lines of code are required
+ 
+    #include "iceoryx_posh/runtime/posh_runtime.hpp"
+
+    iox::runtime::PoshRuntime::initRuntime("/some_unique_application_name");
+
+The application name must be unique among all other applications and have a leading `\`.
+
+
+### Topics
+
+A topic in iceoryx defines the data to be transmitted and is uniquely identified by three string identifiers.
+
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
+
+A triple consisting of such strings is called a ``service description``. In Autosar terminology these three
+identifiers are called ``Service``, ``Instance`` and ``Event`` respectively.
+
+Two topics are considered matching if all these three strings are element-wise equal, i.e. group, instance and topic
+names are the same for both of them.
+
+This means the group and instance identifier can be ignored to create different topics. They will be needed for
+advanced filtering functionality in the future.
+
+The data type of the topic can be an arbitrary C++ class, struct or plain old data type.
+
+### Publisher
+A publisher is tied to a topic and needs a service description to be constructed. If it is typed one needs to
+additionally specify the data type as a template parameter. Otherwise publisher is only aware of raw memory and the
+user has to take care that it is interpreted correctly.
+
+Once it has offered its topic, it is able to publish (send) data of the specific type. Note that it is possible to
+have multiple publishers for the same topic.
+
+### Subscriber
+Symmetrically a subscriber also corresponds to a topic and thus needs a Service Description to be constructed. As for
+publishers we distinguish between typed and untyped subscribers.
+
+Once a subscriber is subscribed to some topic, it is able to receive data of the type tied to this topic. In the
+untyped case this is raw memory and the user must take care that it is interpreted in a way that is compatible to the
+data that was actually send.
+
+When multiple publishers have offered the same topic the subscriber will receive the data of all of them (but in
+indeterminate order between different publishers).
+
+### Waitset
+The easiest way to receie data is to periodically poll whether data is available. This is sufficient for simple use
+cases but inefficient in general, as it often leads to unnecessary latency and wake-ups without data.
+
+The ``Waitset`` can be used to relinquish control (putting the thread to sleep) and wait for user defined conditions
+to become true. Usually these conditions correspond to the availability of data at specific subscribers. This way we
+can (almost) immediately wake up when data is available and will avoid unnecessary wake-ups if no data is available.
+
+To do so it manages a set of triggers which can be activated and indicate that a corresponding condition became true
```

Comment:
Proposal for better readability:
~~To do so~~ It manages a set of triggers which can be activated to indicate that a corresponding condition became true which ~~in turn will~~ wake up a potentially waiting thread.

Reply:
wake**s** up a..

Reply:
Done